packages/fx-core/tests/plugins/resource/aad/unit/generateArmTemplates.test.ts

```diff
@@ -54,6 +54,7 @@ describe("AadGenerateArmTemplates", () => {
       );
       chai.assert.isUndefined(armTemplateResult.Configuration);
       chai.assert.isUndefined(armTemplateResult.Provision);
+      chai.assert.strictEqual(armTemplateResult.SchemaVersion, "0.1.0");
```

Comment:
reference constant instead of using "0.1.0"?

Reply:
test case use the value direct is better.